pkgs/by-name/sm/smctemp/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  darwin,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "smctemp";
+  version = "0.6.0";
+
+  src = fetchFromGitHub {
+    owner = "narugit";
+    repo = pname;
```

```suggestion
    repo = "smctemp";
```
We want to avoid linking repo name to project name, see:
- #277994